source/common/network/dns_impl.cc

```diff
@@ -135,5 +160,13 @@ ActiveDnsQuery* DnsResolverImpl::resolve(const std::string& dns_name, ResolveCb
   }
 }
 
+void DnsResolverImpl::PendingResolution::getHostByName(int family) {
+  ares_gethostbyname(channel_, dns_name_.c_str(),
+                     family, [](void* arg, int status, int timeouts, hostent* hostent) {
+                       static_cast<PendingResolution*>(arg)->onAresHostCallback(status, hostent);
+                       UNREFERENCED_PARAMETER(timeouts);
```

Comment:
i think you should be able to (void *arg, int status, int, hostent* hostent) and remove UNREFERENCED_PARAMETER.